### DISCUSSION TITLE
Potions disappeared - But appears to be start of implementing brewing

### DISCUSSION THREAD

**Discussion by odgregg** [Lurkars/gloomhavensecretariat#312] [Q&A] [answered]
I was part way through logging a bug about potions having disappeared from the items you can craft (work around of unlocking them on the campaign sheet).

Fortunately I checked the changes for the last few releases and it appears you've started working on brewing implementation?

**Comment by Lurkars** [comment, marked as answer]
Yes, the bug was the opposite: all potions were available including it's "recipe". So I just needed to quick fix this. So current progress for unlocking potions: know the potion number and manually unlock it on the campaign sheet.

A brewing implementation will come soon.

**Reply by odgregg** [reply]
Okay, I've just got back to my campaign to continue, and like the new brewing interface (which I understand may not be finished).

However I brewed a new potion, combining two of the same herb and unlocked potion 098, which worked.

I then combined two of another herb (wanting to create another 098 for crafting reasons) and as soon as I click on the second herb the 098 card comes up but with no way to select "Brew" button/link.

I got around it by manually adding a 098 to inventory and removing the two herbs.

**Reply by Lurkars** [reply]
I think there is just the scrollbar missing and the brew button is out of bounce! Will be fixed soon!

**Reply by Lurkars** [reply]
Should be fixed in [v0.78.2](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.78.2).

**Reply by odgregg** [reply]
Thanks! Will test tonight.

**Reply by odgregg** [reply]
Actually, I've just tested it on your hosted version and it's working now.
